SimPEG/potential_fields/gravity/simulation.py

```diff
@@ -312,6 +312,26 @@ def evaluate_integral(self, receiver_location, components, tolerance=1e-4):
         return np.vstack([rows[component] for component in components])
 
 
+class SimulationEquivalentSourceLayer(BaseEquivalentSourceLayerSimulation, Simulation3DIntegral):
+    """
+    Equivalent source layer simulations
+
+    Parameters
+    ----------
+    mesh : discretize.BaseMesh
+        A 2D tensor or tree mesh defining discretization along the x and y directions
+    cell_z_top : numpy.ndarray or float
+        Define the elevations for the top face of all cells in the layer
+    cell_z_bottom : numpy.ndarray or float
+        Define the elevations for the bottom face of all cells in the layer
+    """
+
+    def __init__(self, mesh, cell_z_top, cell_z_bottom, **kwargs):
+
+        BaseEquivalentSourceLayerSimulation.__init__(self, mesh, cell_z_top, cell_z_bottom)
```

Comment:
These two individual calls to the parent classes should both be done with a single call to `super().__init__`, just pass everything by keyword and it should all end up in the correct place

Reply:
actually... with the way this is written, I bet you could just delete the `__init__` functions on this class and it would still work just fine, as the `BaseEquivalentSourceLayerSimulation` class's `__init__` should be at the top of the inheritance.
(Then that classes call to `super` would go up a level)

Reply:
I concur

Reply:
I removed the redundant __init__ functions as suggested.